tests/basic/test-plugin-manager.cpp

```diff
@@ -60,7 +60,11 @@ void load_plugin_from_command_line()
 void load_plugin_from_file()
 {
     std::ofstream config( PluginManager::configuration_file );
+#ifdef RINGMESH_DEBUG
+    config << "RINGMesh_iod";
```

Comment:
Or something like:
config << "RINGMesh_io";
#ifdef RINGMESH_DEBUG
    config << "d";
#endif

I find it easier to read